maven_iuvs/graphics/line_fit_plot.py

```diff
@@ -50,72 +75,100 @@ def detector_image(myfits, integration=0,
 
     cmap = idl_colorbars.getcmap(cmap)
 
-    ax.set_xlim([0, 1024])
-    ax.set_ylim([0, 1024])
```

Comment:
the original intent here was to plot the full extent of the detector and overplot the file data range... this functionality needs to be preserved.

Reply:
Fixed by 19ff4ea22866d14108ef61c8ecf5b25a8d7f81d7